doc/api/buffer.md

```diff
@@ -1482,12 +1482,36 @@ calls can be chained.
 ```js
 const buf = Buffer.from([0x1, 0x2, 0x3, 0x4, 0x5, 0x6, 0x7, 0x8]);
 console.log(buf);
-  // Prints Buffer(0x1, 0x2, 0x3, 0x4, 0x5, 0x6, 0x7, 0x8)
+  // Prints <Buffer 01 02 03 04 05 06 07 08>
 buf.swap32();
 console.log(buf);
-  // Prints Buffer(0x4, 0x3, 0x2, 0x1, 0x8, 0x7, 0x6, 0x5)
+  // Prints <Buffer 04 03 02 01 08 07 06 05>
 ```
 
+### buf.swap64()
+<!-- YAML
+added: v6.x.x
```

Comment:
Using `added: REPLACEME` will lead the person doing the release that contains this feature to fill in the correct version.